core/controllers/blog_admin.py

```diff
@@ -57,6 +57,8 @@ class BlogAdminHandlerNormalizedPayloadDict(TypedDict):
     config_property_id: Optional[str]
 
 
+# TODO(#16364): Refactor 'BlogAdminHandler' so that we don't have to use action
```

Comment:
Can you talk to @Rijuta-s about this? If this is what needs to be done then she should probably resolve it. /cc @gp201

Reply:
A email thread is sent to rijuta
Thanks!

Reply:
OK -- though I'm not approving until this is resolved since we need clarity on whether this TODO is appropriate or not. (In general I would suggest flagging new TODOs for reviewers at the outset unless they are part of an already-established pattern.)